src/read_utils.jl

```diff
@@ -2,6 +2,37 @@ using DelimitedFiles
 
 # ---------------------------------------------------------------------------------------- #
 
+function make_symmetryvector(bandirsd::Dict{String, Vector{Pair{UnitRange{Int64}, Vector{Int64}}}}, 
```

Comment:
Guess this should be plural? Maybe `build_symmetryvectors`?
```suggestion
function build_symmetryvectors(bandirsd::Dict{String, Vector{Pair{UnitRange{Int}, Vector{Int}}}}, 
```